esi/include.go

```diff
@@ -52,6 +53,12 @@ func (i *includeTag) Process(b []byte, req *http.Request) ([]byte, int) {
 		return nil, len(b)
 	}
 
+	if strings.HasPrefix(i.src, "//") {
+		i.src = req.URL.Scheme + i.src
```

Comment:
This looks like it should work, but for some reason `req` seems to be empty when calling this.